crates/ruff_python_parser/src/string.rs

```diff
@@ -220,7 +227,7 @@ impl StringParser {
                 if self.kind.is_byte_string() && !first_char.is_ascii() {
                     return Err(LexicalError::new(
                         LexicalErrorType::InvalidByteLiteral,
-                        self.range(self.get_pos()),
+                        TextRange::empty(self.position()),
```

Comment:
Actually, I don't even think this code is reachable because we check for non-ascii literals at the start of parsing the byte literals:

https://github.com/astral-sh/ruff/blob/12368a29ff3cb16010de090d9537d0e7b384a9fa/crates/ruff_python_parser/src/string.rs#L330-L335

I'll remove this.